### PR TITLE
SFR-2073: Added local S3 docker container via localstack

### PR DESCRIPTION
**Pull request by kylevillegas93**
## Description

- This change adds a local docker container for S3 and uses localstack to create the buckets

## Testing
`docker compose up` -- tested by trying to get object from S3 in dev setup script

### CODE REVIEW

**Review by mitri-slory — APPROVED** [review]
The local S3 Docker container looks great. I only have one comment but otherwise this is ready to go.

File: config/sample-compose.yaml [hunk 1]
```diff
@@ -42,9 +42,11 @@ HATHI_API_ROOT: https://babel.hathitrust.org/cgi/htd
 #HATHI_API_SECRET: 
 
 # AWS CONFIGURATION
-#AWS_ACCESS:
-#AWS_SECRET:
-#AWS_REGION:
+AWS_ACCESS: test
+AWS_SECRET: test
+AWS_REGION: us-east-1
+FILE_BUCKET: drb-files-local
```
[the remark this answers is not in the document]
Comment (kylevillegas93) [reply config/sample-compose.yaml:48]: Just docker!

**Comment by Apophenia** [comment]
Great addition, thank you for this update!

File: config/sample-compose.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (mitri-slory) [reply config/sample-compose.yaml:48]: Appreciate the clarification.

**Pull request merged**